src/types/emotion.d.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference types="@emotion/react/types/css-prop" />
+import '@emotion/react';
+
+import { ThemeType } from '../constants/themes/types';
+
+declare module '@emotion/react' {
+  // eslint-disable-next-line @typescript-eslint/no-empty-interface
+  export interface Theme extends ThemeType {}
```

Comment:
Do you know if this will have any influence on how we use `ThemeType` on the Gatsby sites?

Specifically [here](https://github.com/curology/curology-gatsby/blob/main/src/types/modules/%40emotion/styled/index.ts) and [here](https://github.com/curology/curology-gatsby/blob/main/src/types/modules/emotion-theming/index.ts)

Reply:
Yep, we no longer need those files (this PR removes those equivalents here). This file will be the new standard: https://emotion.sh/docs/typescript#define-a-theme